Azure/azure-cli
# [App Service] Add support for vnet route all setting

**Description**
Adding support for vnet route all setting in config set
Defaulting to vnet_route_all in webapp vnet-integration add
Add support for skipping delegation in webapp vnet-integration add (for scenarios with limited permission on subnet)
Adding support for cross-subscription vnet-integration
Fixing minor syntax error in help file

**Testing Guide**
az webapp config set -g <group> -n <webapp> --vnet-route-all-enabled
az webapp config set -g <group> -n <webapp> --vnet-route-all-enabled false

az webapp vnet-integration add -g <group> -n <webapp> --vnet <vnet-name> --subnet <subnet> [--skip-delegation-check]

**History Notes**


[AppService] az webapp config set: Add support for VNet Route All property.
[AppService] az webapp vnet-integration add: Default to VNet Route All. Allow cross subscription integration.

---

This checklist is used to make sure that common guidelines for a pull request are followed.
